modules/custom/activity_basics/activity_basics.module

```diff
@@ -14,13 +15,32 @@ use Drupal\group\Entity\GroupContent;
  */
 function activity_basics_entity_insert(EntityInterface $entity) {
```

Comment:
I'd suggest adding a return type hint and deleting an error from the list of ignored by PHPStan.

Reply:
Fixed